tests/test_dashboards.py

```diff
@@ -37,15 +41,26 @@ def test_dashboard_management(self):
         self.assertDictEqual(resp_create, resp_get)
 
     def test_get_dashboard(self):
-        dashboard = self.api.describe_dashboard('pylana-test-log-.*')
-        self.assertEqual(dashboard.get('title'), 'pylana-test-log-from-df')
+        dashboard = self.api.describe_dashboard('incident.*')
+        self.assertEqual(dashboard.get('title'), 'incident-test-dashboard')
 
         with self.assertRaises(Exception):
             _ = self.api.get_dashboard('never-ever-matches-a-dashboard')
 
     def test_get_dashboard_id(self):
-        dashboard_id = self.api.get_dashboard_id('pylana-test-log-from-df')
+        dashboard_id = self.api.get_dashboard_id('incident-test-dashboard')
         self.assertIsInstance(dashboard_id, str)
 
         with self.assertRaises(Exception):
             _ = self.api.get_dashboard_id('never-ever-matches-a-dashboard')
+
+    def test_share_dashboard(self):
+        id_ = self.api.get_dashboard_id('incident-test-dashboard')
+
+        actual = self.api.share_dashboard(id_,
+                                                  [self.api.user.organization_id]).json()
+
+        expected = {'sharing': {'numFailures': 0, 'numSuccesses': 1},
+                            'unsharing': {'numFailures': 0, 'numSuccesses': 0}}
+
+        self.assertDictEqual(actual, expected)
```

Comment:
here the ondenting can be improved for readbility. We shoould really look into [black](https://github.com/psf/black).